docs/user/user/pings/custom.md

```diff
@@ -41,6 +41,10 @@ search:
     - http://example.com/path/to/data-review
 ```
 
+{{#include ../../../shared/blockquote-info.html}}
+
+##### Reserved ping names
+
 > Note: the names `baseline`, `metrics`, `events`, `deletion-request` and `all-pings` are reserved and may not be used as the name of a custom ping.
 
 ## Loading custom ping metadata into your application or library
```

Comment:
`registerPings` should be documented under the General API section on the API Reference, as it is part of the Glean General API.